File: C++/896. Monotonic Array.cpp
class Solution {
public:
    bool isMonotonic(vector<int>& n) {
        int i,f=0;
        if(n.size()<3)
        return true;
        for(i=0;i<n.size()-1;i++)
        {
            if(n[i]<=n[i+1])
            f=1;
            else{
                f=0;
            break;
            }
        }
        if(f==0){
        for(i=0;i<n.size()-1;i++)
        {
            if(n[i]>=n[i+1])
            f=1;
            else
            {
                f=0;
                break;
            }
        }}
        return (f==1);
    }
};

File: JAVA/896. Monotonic Array.java
class Solution {
    public boolean isMonotonic(int[] n) {
        int a=1,b=1,i;
        for(i=0;i<n.length-1;i++)
        {
            if(n[i]<n[i+1])
            a=0;
            if(n[i]>n[i+1])
            b=0;
        }
        return(a==1||b==1);
    }
}